index.html

```diff
@@ -0,0 +1,61 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
```

Comment:
- [ OPTIONAL ] Please make sure that you set the appropriate padding button for your header as it is in the Figma design.

### Your design
![image](https://user-images.githubusercontent.com/92303891/234993825-af6c288c-ef35-42fe-a194-4d32a4d92c86.png)
  
  ### Figma Design
  ![image](https://user-images.githubusercontent.com/92303891/234994393-8921e42d-e568-42b4-a99d-56ada06783e5.png)